### PR TITLE
Use a capsule service provider.

### PR DESCRIPTION
**Pull request by imkitae**
Issue: https://github.com/ridibooks/cms/issues/18

This uses `jguyomard/silex-capsule-eloquent` package, a service provider capsulizing database.

### CODE REVIEW

File: src/CmsServerApplication.php [hunk 1]
```diff
@@ -31,28 +31,35 @@ public function __construct(array $values = [])
...
+                        'host' => $mysql['host'],
+                        'database' => $mysql['database'],
+                        'username' => $mysql['user'],
+                        'password' => $mysql['password'],
+                        'charset' => 'utf8',
+                        'collation' => 'utf8_unicode_ci',
+                        'prefix' => '',
+                        'options' => [
+                            // mysqlnd 5.0.12-dev - 20150407 에서 PDO->prepare 가 매우 느린 현상
+                            \PDO::ATTR_EMULATE_PREPARES => true
```
Comment (namenu) [inline src/CmsServerApplication.php]: Do we still need this after the migration?

File: src/CmsServerApplication.php [hunk 2]
```diff
@@ -31,28 +31,35 @@ public function __construct(array $values = [])
...
+                'capsule.connections' => [
+                    'default' => [
+                        'driver' => 'mysql',
+                        'host' => $mysql['host'],
+                        'database' => $mysql['database'],
+                        'username' => $mysql['user'],
+                        'password' => $mysql['password'],
+                        'charset' => 'utf8',
+                        'collation' => 'utf8_unicode_ci',
+                        'prefix' => '',
```
Comment (namenu) [inline src/CmsServerApplication.php]: What's this `prefix` for?

File: src/CmsServerApplication.php [hunk 3]
```diff
@@ -31,28 +31,35 @@ public function __construct(array $values = [])
...
-            'prefix' => '',
-            'options' => [
-                // mysqlnd 5.0.12-dev - 20150407 에서 PDO->prepare 가 매우 느린 현상
-                \PDO::ATTR_EMULATE_PREPARES => true
+        $this->register(
+            new CapsuleServiceProvider(),
+            [
+                'capsule.connections' => [
+                    'default' => [
+                        'driver' => 'mysql',
```
Comment (namenu) [inline src/CmsServerApplication.php]: `$mysql` variable is redundant. (Assign them directly)

File: src/CmsServerApplication.php (hunk 2 above)
Comment (imkitae) [inline src/CmsServerApplication.php]: This is for table prefix, but we defined full table names at each model. I'll delete this.

File: src/CmsServerApplication.php (hunk 1 above)
Comment (imkitae) [inline src/CmsServerApplication.php]: No, we don't I think.. (mariaDB version we use supports prepares statement.) Let's remove this.

**Pull request merged**